prescient/simulator/oracle_manager.py

```diff
@@ -325,10 +325,14 @@ def _report_sced_stats(self, ops_stats: OperationsStats):
         if ops_stats.over_generation!= 0.0:
             print("Over-generation reported at t=%d -   total=%12.2f" % (1, ops_stats.over_generation))
 
-        if ops_stats.reserve_shortfall != 0.0:
-            print("Reserve shortfall reported at t=%2d: %12.2f" % (1, ops_stats.reserve_shortfall))
-            print("Quick start generation capacity available at t=%2d: %12.2f" % (1, ops_stats.available_quickstart))
-            print("")
+        try:
+            for res in ops_stats.reserve_products:
+                if ops_stats.reserve_shortfalls[res] != 0.0:
+                    print("Reserve shortfall reported at t=%2d: %12.2f" % (1, ops_stats.reserve_shortfalls[res]))
+                    print("Quick start generation capacity available at t=%2d: %12.2f" % (1, ops_stats.available_quickstart))
+                    print("")
+        except Exception as ex:
+            print(ex)
```

Comment:
Reasoning for this `try/except` block? Doesn't generally seem like it should raise an exception.